src/SurveyDesign.jl

```diff
@@ -71,41 +116,57 @@ struct StratifiedSample <: AbstractSurveyDesign
 end
 
 # `show` method for printing information about a `StratifiedSample` after construction
-function Base.show(io::IO, design::StratifiedSample)
-    printstyled("Stratified Sample:\n")
+function Base.show(io::IO, ::MIME"text/plain", design::StratifiedSample)
+    printstyled("Stratified Sample:\n"; bold = true)
     printstyled("data: "; bold = true)
-    print(size(design.data)[1], "x", size(design.data)[2], " DataFrame")
+    print(size(design.data, 1), "x", size(design.data, 2), " DataFrame")
+    printstyled("\nweights: "; bold = true)
+    print_short(design.data.weights)
     printstyled("\nprobs: "; bold = true)
     print_short(design.data.probs)
     printstyled("\nstrata: "; bold = true)
     print_short(design.data.strata)
-    # TODO: change fpc
     printstyled("\nfpc: "; bold = true)
-    print("\n    popsize: ")
-    print_short(design.data.popsize)
-    print("\n    sampsize: ")
-    print_short(design.data.sampsize)
+    print_short(design.fpc)
+    printstyled("\n    popsize: "; bold = true)
+    print(design.popsize)
+    printstyled("\n    sampsize: "; bold = true)
+    print(design.sampsize)
 end
 
 """
-A `ClusterSample` object holds information necessary for surveys sampled by
-clustering.
+    ClusterSample <: AbstractSurveyDesign
```

Comment:
```suggestion
    `ClusterSample <: AbstractSurveyDesign`
```

Reply:
The back ticks aren't necessary. See for example this source code: https://github.com/JuliaLang/julia/blob/5544a0fab7648cfa61fe79cd557a7504a92ec1b5/base/docs/basedocs.jl#L1775-L1779